### ISSUE TITLE
@types/express-validator  with compile errors.

### ISSUE THREAD

**Issue by diegolaciar** [DefinitelyTyped/DefinitelyTyped#15355]
Hi all,

I installed last version of "@types/express-validator": "^2.20.33" and get this errors on TS compile.
```
0] 23  export = ExpressValidator;
[0]     ~~~~~~~~~~~~~~~~~~~~~~~~~~
[0] 
[0] node_modules/@types/express-validator/index.d.ts(23,2): error TS2300: Duplicate identifier 'export='.
[0] 
[0] 
[0] 37  interface Dictionary<T> { [key: string]: T; }
[0]                               ~~~~~~~~~~~~~~~~~
[0] 
[0] node_modules/@types/express-validator/index.d.ts(37,28): error TS2374: Duplicate string index signature.
[0] 
[0] 
[0] 24   export = ExpressValidator;
[0]      ~~~~~~~~~~~~~~~~~~~~~~~~~~
[0] 
[0] node_modules/express-validator/index.d.ts(24,3): error TS2300: Duplicate identifier 'export='.
[0] 
[0] 
[0] 9 import * as expressValidator from 'express-validator';
[0]                                     ~~~~~~~~~~~~~~~~~~~
[0] 
[0] src/backend/app.ts(9,35): error TS2497: Module '"express-validator"' resolves to a non-module entity and cannot be imported using this construct.
```

Anyone can help with this definition ?

Regards,
Diego

**Comment by sundriver** [comment]
+1

**Comment by pumano** [comment]
Have that problem too.

Workaround:
1) delete from node_modules @types/express-validator
2) delete @types/express-validator dependency from package.json 

and just use express-validator: 
`import expressValidator = require('express-validator');`

**Comment by jpap** [comment]
I had the same issues; it looks like [express-validator](https://github.com/ctavan/express-validator/commits/master/index.d.ts) now has typings built-in as of December 2016.

The proper solution is to remove `@types/express-validator` from your project.

**Comment by micksatana** [comment]
@diegolaciar please help close this issue. It was solved by PR #16633

**Comment by diegolaciar** [comment]
Looks it is solved. Close this issue as requested by @micksatana
